### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by darkvergus**
- Removed redundant code that the compiler auto generates.
- Fixed lambda_methods.
- General Cleanup

### CODE REVIEW

**Review by AngeloTadeucci — APPROVED** [review]

File: MapleServer2/Enums/SystemNotice.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace MapleServer2.Enums;
 
-public enum SystemNotice : int
+public enum SystemNotice
```
Comment (Zintixx) [inline MapleServer2/Enums/SystemNotice.cs:3]: Like this and many others, has this PR even been tested? Without indicating it's an int, it's going to ruin a lot of packets being written. How will the server know when to write an int or not?

**Review by Zintixx — CHANGES_REQUESTED** [review]

File: MapleServer2/Enums/SystemNotice.cs (hunk 1 above)
Comment (darkvergus) [reply MapleServer2/Enums/SystemNotice.cs:3]: by default its an int

Comment (AngeloTadeucci) [reply MapleServer2/Enums/SystemNotice.cs:3]: Yes this has been tested and Rider says this: 'int' is default enum governing type

Comment (darkvergus) [reply MapleServer2/Enums/SystemNotice.cs:3]: the compiler assumes its an int by default thats why its ok to remove it

**Review by Zintixx — APPROVED** [review]

**Review by AlanMorel — APPROVED** [review]

**Pull request merged**